lib/octokit/client/deployments.rb

```diff
@@ -1,69 +1,87 @@
 module Octokit
   class Client
-
     # Methods for the Deployments API
     #
-    # @see https://developer.github.com/v3/repos/commits/deployments/
+    # @see https://developer.github.com/v3/repos/deployments/
     module Deployments
-
-      # Fetch a single deployment for a repository
+      # Get a single deployment
       #
       # @param repo [Integer, String, Repository, Hash] A GitHub repository
-      # @param deployment_id [Integer, String, Repository, Hash] A GitHub repository
+      # @param deployment_id [Integer] The ID of the deployment
```

Comment:
This line was just plain wrong, as well.

The routes file's description for this field is an empty string, since it's not described in the docs site (this is a field that is part of the URL). I made a fallback description for `*_id` fields that are empty "the ID of the _$whatever_".